### PR TITLE
Add form

### PR DESCRIPTION
**Pull request by SamTush**
## What does this PR do?

This PR is raised to implement the following features: 👍 :

- [x] Add form

#### Thanks for reviewing my PR 😊

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -1,69 +0,0 @@
-class CommentsController < ApplicationController
```
Comment (tugsem) [inline app/controllers/comments_controller.rb]: - [ ] Please add the comments_controller.rb into your project directory with the required merhods in your routes.rb file as per the requirement.

File: app/controllers/likes_controller.rb [hunk 2]
```diff
@@ -1,69 +0,0 @@
-class LikesController < ApplicationController
```
Comment (tugsem) [inline app/controllers/likes_controller.rb]: - [ ]  Please add the likes_controller.rb into your project directory back with the required methods as in your routes, as per the requirement.

File: app/views/posts/show.html.erb [hunk 3]
```diff
@@ -1,10 +1,46 @@
...
+          <% if @comments.any?  %>
+            <% @comments.each do |comment| %>
+              <li>
+                <p><%= User.find_by(id: comment.user_id).name %>: <%= comment.text %></p>
+              </li>
+            <% end %>
+          <% else %>
+            <li>No comments added yet...</li>
+          <% end %>
+        </ul>
```
Comment (tugsem) [inline app/views/posts/show.html.erb]: - [ ] As per the requirement, you should add a button here that redirect to comment/new page with form. And every comment should belong to currrent_user. 
image

File: app/views/posts/show.html.erb [hunk 4]
```diff
@@ -1,10 +1,46 @@
...
+    <div class="single-post-container">
+      <%# user post start %>
+      <article class="post-article">
+        <div class="info">
+          <div>
+            <h2 class="title"><%= @post.title %></h2>
+            <p class="user-name">by <%= @user.name %></p>
+          </div>
+          <p class="posts-counter">
+            Comment: <%= @post.comment_counter || 0 %>, Likes: <%= @post.likes_counter || 0 %>
```
Comment (tugsem) [inline app/views/posts/show.html.erb]: - [ ] As per the requirement, there should be a button to create likes for a post. Kindly implement that. 
image

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @SamTush 👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️posts controller created with create and new actions
✔️current_user method is added
✔️forms are created for posts and allows to add a post by the current user
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SamTush** [comment]
Thank you @tugsem for reviewing my PR👍🏽 Cheer and happy coding 😎

**Review by mwanawabangona — APPROVED** [review]
Hi @SamTush 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job Implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**